spec/migration_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+require 'yaml'
+load File.expand_path(File.join(__FILE__, '../../Rakefile'))
+
+describe 'migrations' do
+  %w(katello katello-devel capsule).each do |scenario|
+
+    context "on #{scenario}" do
+      let(:answers) { File.expand_path(File.join(CONFIG_DIR, "#{scenario}-answers.yaml")) }
+      let(:config) { File.expand_path(File.join(CONFIG_DIR, "#{scenario}.yaml")) }
+
+      before(:each) do
+        ENV['INSTALLER_SCENARIO'] = scenario
+      end
+
+      after(:each) do
+        `git reset HEAD --hard`
```

Comment:
Hm, good point. I'll just fix it so the test migrations don't dump their answers back into the git repo, which is the only reason I have the reset.